NEWS.md

```diff
@@ -1,5 +1,9 @@
 # pkgdown (development version)
 
+* `document` in `build_site()` and `build_reference()` is now defunct after being deprecated in pkgdown 1.6.0. `devel` should be used instead.
+
```

Comment:
FYI in pkgdown I'm trying out a new style where we don't add any line breaks until we do the final release tidy up.